test/blackbox-tests/test-cases/lazy-and.t

```diff
@@ -0,0 +1,28 @@
+Running `dune fmt` when a dependency is not available should not crash this way.
+  $ cat > dune-project <<EOF
+  > (lang dune 3.13)
+  > EOF
+
+  $ cat > dune << EOF
+  > (test
+  >  (name foo)
```

Comment:
does the test work when you remove the `enabled_if` part? if not, it would be useful to add an empty `foo.ml`, to make sure that the PR fixing the issue will actually make the test pass.

Reply:
I added a second dune file without the enabled_if part. In that case everything works as intended (I think)